src/ontology/mondo-edit.obo

```diff
@@ -441658,7 +441660,7 @@ synonym: "PNH" EXACT ABBREVIATION [Orphanet:447]
 xref: DOID:0060284 {source="MONDO:equivalentTo"}
 xref: GARD:0007337 {source="MONDO:equivalentTo"}
 xref: HGNC:8957 {source="GARD:0007337"}
-xref: HP:0004818 {source="MONDO:otherHierarchy", source="DOID:0060284"}
+xref: HP:0004818 {source="MONDO:subClassOf", source="MONDO:otherHierarchy", source="DOID:0060284"}
```

Comment:
I believe the MONDO:subClassOf is only meant for Mondo classes, but I understand your thought process. I think in this case, we would just delete the HPO xref here and move it to the parent class (MONDO_01002443 'paroxysmal nocturnal hemoglobinuria'). Add a seeAlso to the ticket and then we'll have the history of this HPO xref being on this term.